### PR TITLE
Various updates

### PR DESCRIPTION
**Pull request by JensTrier**
Various updates to cover updated syntax for cmdlets and MoH for LBR users.

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9

File: skype/skype-ps/skype/Get-CsOnlinePSTNGateway.md [hunk 1]
```diff
@@ -42,10 +41,10 @@ This example shows all SBCs paired with the tenant.
 
 ### Example 2
 ```powershell
-PS C:\> Get-CsOnlinePSTNGateway | ?{$_.Identity -like "*.contoso.com"}
+PS C:\> Get-CsOnlinePSTNGateway -Filter "*.contoso.com"
 ```
 
-This example selects all SBC with names matching the pattern *.contoso.com. For example: sbc1.contoso.com, sbc2.contoso.com etc
+This example selects all SBC with Identity matching the pattern *.contoso.com. For example: sbc1.contoso.com, sbc2.contoso.com etc
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlinePSTNGateway.md:47]:
```suggestion
This example selects all SBC with Identity matching the pattern *.contoso.com. For example: sbc1.contoso.com, sbc2.contoso.com, etc.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @JensTrier, kindly commit the suggested minor change. Many thanks.

**Comment by JensTrier** [comment]
> Copy review - @JensTrier, kindly commit the suggested minor change. Many thanks.

@JohanFreelancer9 Thanks & done!

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by dariomws — APPROVED** [review]

**Comment by yogkumgit** [comment]
@get-itips @tseward

**Review by tseward — APPROVED** [review]

**Review by get-itips — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**